### PR TITLE
Add pixelset search

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Add a new search field to the `PixelSetListView`.

## Proposal

The current search implementation works as a filter that will look for matching:

* Omics Units reference identifier in pixels
* keywords in Analysis's description
* keywords in Experiment's description

## Sneak peek

screenshot-2018-1-25 pixel sets

### CODE REVIEW

File: apps/explorer/tests/test_views.py [hunk 1]
```diff
@@ -358,3 +359,129 @@ def test_filters_with_no_match(self):
...
+        #
+        # Lorem ipsum source:
+        # http://shooshee.tumblr.com/post/212964026/klingon-lorem-ipsum
+        first_analysis = factories.AnalysisFactory(
+            description=(
+                'Qapla. Dah tlhingan hol mu ghom a dalegh. Qawhaqvam '
+                'chenmohlu di wiqipe diya ohvad ponglu. Ach jinmolvamvad '
+                'Saghbe law tlhingan hol, dis, oh mevmohlu.'
+            )
+        )
```
Comment (willdurand) [inline apps/explorer/tests/test_views.py]: 👍

File: apps/explorer/tests/test_views.py [hunk 2]
```diff
@@ -358,3 +359,129 @@ def test_filters_with_no_match(self):
...
+        self.assertContains(
+            response,
+            (
+                '<td class="filename">'
+                '<!-- Pixel set file name -->'
+                '{}'
+                '</td>'
+            ).format(
+                filename(first_pixel_set.pixels_file.name)
+            ),
```
Comment (willdurand) [inline apps/explorer/tests/test_views.py]: I think a f-string would be much easier to read. I had to read this assertion three times

Comment (jmaupetit) [reply apps/explorer/tests/test_views.py]: I have to confess that I am not used to f-strings…

**Comment by jmaupetit** [comment]
Updated!

**Pull request merged**